content/en/get-started/cards/docker-691.md

```diff
@@ -0,0 +1,15 @@
+### CDAP 6.9.1 for Docker
+
+#### Description
+
+CDAP Docker provides a clean, contained environment for trying out CDAP on your desktop or laptop.
+
+####  What's New:
+
+* View [release notes](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/1888059393/CDAP+Release+6.9.1)
```

Comment:
@CuriousVini is this correct?

Reply:
Use this one

https://cdap.atlassian.net/wiki/spaces/DOCS/pages/1902084097/CDAP+Release+6.9.1